app/subscriber/src/components/date-filter/DateFilter.tsx

```diff
@@ -1,78 +1,58 @@
-import moment from 'moment';
+import moment, { Moment } from 'moment';
 import React from 'react';
 import ReactDatePicker from 'react-datepicker';
 import { FaCalendarDay, FaCaretLeft, FaCaretRight } from 'react-icons/fa6';
-import { IFilterSettingsModel } from 'tno-core';
 
 import * as styled from './styled';
 
 export interface IDateFilterProps {
-  filter: IFilterSettingsModel;
-  storeFilter: (filter: IFilterSettingsModel) => void;
-  onChangeDate?: (filter: IFilterSettingsModel) => void;
+  date: string | Date | Moment | undefined;
```

Comment:
We no longer mutate parent state within this component.  The component now correctly only cares about the start date.